Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -31,39 +31,134 @@
 #include "VideoCommon/VideoCommon.h"
 #include "VideoCommon/VideoConfig.h"
 
-namespace OGL
+constexpr const char* GLSL_PROGRAM_VS = R"GLSL(
```

Comment:
These should still be within the `OGL` namespace. Making them regular string arrays (i.e. `constexpr char name[] = ...`) also shortens the executable size a very miniscule amount, since the compiler only needs to store the string data itself into the executable (and not a pointer address as well for each pointer to point to said stored string data).